### DISCUSSION TITLE
Renovate PRs don't group updates together - monorepo

### DISCUSSION THREAD

**Discussion by shady00999** [renovatebot/renovate#26669] [Request Help] [answered]
### How are you running Renovate?

None

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

I want to get Renovate to open one PR for `@types/jest`, for example, that should include all of the monorepo projects that include this package and update it across all projects in the same PR. Right now Renovate opens a PR for each of the monorepo projects individually - if 5 projects include the package then 5 PRs will be opened. Is there something I'm missing with the following config? 
I tried to add the individual packages into `excludePackageNames` property on packagesRules for the ones that use `matchFileNames`, but it doesn't change anything. I also tried to use 
`"matchFileNames": [ "**/package.json"]` on the individual package rules so that `matchFileNames` would be overridden for these rules, but also didn't work.

```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "github>renovatebot/.github"
  ],
  "assignees": [],
  "addLabels": [
    "dependencies"
  ],
  "includePaths": [
    "monorepo/project1/*",
    "monorepo/project2/*",
    "monorepo/project3/*",
    "monorepo/project4/*",
    "monorepo/project5/*"
  ],
  "enabledManagers": [
    "npm",
    "dockerfile",
    "github-actions"
  ],
  "baseBranches": [
    "dev"
  ],
  "prConcurrentLimit": 5,
  "enabled": true,
  "prHourlyLimit": 2,
  "minimumReleaseAge": "14 days",
  "automerge": false,
  "automergeType": "pr",
  "prCreation": "immediate",
  "rebaseWhen": "conflicted",
  "timezone": "Europe/London",
  "dependencyDashboard": true,
  "semanticCommits": "disabled",
  "additionalBranchPrefix": "{{parentDir}}-",
  "constraints": {
    "node": "20.10.0"
  },
  "dockerfile": {
    "semanticCommitType": "build"
  },
  "packageRules": [
    {
      "groupName": "Projects1",
      "matchFileNames": [
        "monorepo/project1/*"
      ],
      "commitMessagePrefix": "Renovate projects1"
    },
    {
      "groupName": "Projects2",
      "matchFileNames": [
        "monorepo/project2/*",
        "monorepo/project3/*",
        "monorepo/project4/*"
      ],
      "commitMessagePrefix": "Renovate projects2"
    },
    {
      "groupName": "Projects3",
      "matchFileNames": ["monorepo/project5/*"],
      "commitMessagePrefix": "Renovate projects3"
    },
    {
      "matchPackageNames": [
        "@types/jest"
      ],
      "groupName": "jest monorepo"
    },
    {
      "matchPackageNames": [
        "lodash"
      ],
      "groupName": "lodash monorepo"
    }
  ]
}

```

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Renovate defaults to the behavior you want. That default behavior has been changed by this option:

```
  "additionalBranchPrefix": "{{parentDir}}-",
```

Essentially that option is saying "split updates by parent directory". You should check which colleague made that setting and if they agree with you to remove it

**Reply by shady00999** [reply]
Thanks for the fast reply and help. The solution works.
